README.md

```diff
@@ -456,3 +456,13 @@ redef Kafka::kafka_conf = table( ["metadata.broker.list"] = "node1:6667"
 ## Contributing
 
 If you are interested in contributing to this plugin, please see our [CONTRIBUTING.md](.github/CONTRIBUTING.md).
+
+### Releases
+
```

Comment:
Should mention *who* can do this.  I think it may be worth it's own file, like Maintainers.md

Reply:
IE> this isn't a contributors thing

Reply:
I'm fine documenting things better, will do